dora/core/server/master/src/main/java/alluxio/master/job/DoraLoadJob.java

```diff
@@ -408,6 +413,15 @@ public List<DoraLoadTask> getNextTasks(Collection<WorkerInfo> workers) {
     List<DoraLoadTask> list = new ArrayList<>();
     if (mPreparingTasks.compareAndSet(false, true)) {
       try {
+        Iterator<DoraLoadTask> it = mRetryTaskList.iterator();
+        if (it.hasNext()) {
```

Comment:
I feel like we want to return a list of tasks with the number of works？

Reply:
i would like to be conservative here as the reason of tasks in mRetryTaskList is bcos of overload situation so i would like to poll out 1 task first if there's tasks needed retry

Reply:
nit: add a comment to understand why we only pull one task